correlation_study.py

```diff
@@ -63,16 +63,35 @@ def get_correlated_stocks(self, use_pct_change=False):
             for j in range(len(self.tickers)):
                 if use_pct_change:
                     # Calculate percentage change
-                    vals_i = self.dataframe[self.tickers[i]].pct_change().dropna().to_numpy()
-                    vals_j = self.dataframe[self.tickers[j]].pct_change().dropna().to_numpy()
+                    vals_i = self.dataframe.df[self.tickers[i]].pct_change().dropna().to_numpy()
```

Comment:
You can just use df or dataframe, why dataframe.df?

Reply:
No, actually I can't ;/

Reply:
Why? Please change the code so you don't have to use dataframe.df